### DISCUSSION TITLE
[1606180]下一站江湖Ⅱ {全DLC+成就}

### DISCUSSION THREAD

**Discussion by wawmb** [pjy612/SteamManifestCache#907] [Share]
### 游戏中文名
下一站江湖Ⅱ

`appid:1606180`
### 密钥列表
keys:
```
1606181;78085baa1a1489cac9f2593fc792421a3c773edf3317721f5b02ccb7b9c07898
2905470;5e0e8e4e9b25bd36951858ffd7fe45086c92ca50d89481201f01b39e18aff304
```
email;1642348878@qq.com

### 清单附件

[下一站江湖2全DLC+成就.zip](https://github.com/user-attachments/files/17673187/2.DLC%2B.zip)

### 备注

新人冒泡~

**Comment by pjy612** [comment]
* 密钥格式 不对
* 游戏ID 不对
* #26

**Reply by wawmb** [reply]
感谢回复，已按模板更新格式 ，如还有问题，请指出
https://store.steampowered.com/app/1606180/II/

**Reply by pjy612** [reply]
问题密钥格式还是不对 ╮(￣▽￣")╭

**Reply by wawmb** [reply]
确实没注意 中间已修改为分号;
